ddtrace/commands/ddtrace_run.py

```diff
@@ -1,11 +1,22 @@
 #!/usr/bin/env python
+import argparse
 from distutils import spawn
 import os
 import sys
 import logging
 
-debug = os.environ.get('DATADOG_TRACE_DEBUG')
-if debug and debug.lower() == 'true':
+import ddtrace
+from ddtrace.compat import PY2
+
+
+if PY2:
+    # Python 2 does not have PermissionError but Python 3 does.
+    class PermissionError(Exception):  # noqa
```

Comment:
should this be in `compat`?

`from ddtrace.compat import PermissionError` ?

Reply:
I'd rather not as this isn't a pattern we'd want to follow in most cases. It's just a workaround since the exception doesn't exist. It just prevents a NameError and then we do additional checking for Python 2. By including it in compat it might appear as though Python 2 actually offers a PermissionError which is not true (and probably why six doesn't include it)